covalent/_shared_files/schemas/asset.py

```diff
@@ -28,11 +28,13 @@ class AssetSchema(BaseModel):
     remote_uri: Optional[str] = None
 
     # Size of the asset in bytes
-    size: Optional[int] = 0
+    size: int
```

Comment:
I see that size is not optional anymore. Just wanted to double check that you did a grep to check if all the sizes are set.